javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -205,6 +205,25 @@ public static String getSimpleName(ExecutableElement element) {
         return sj.toString();
     }
 
+    /**
+     * Returns a user-friendly name for the given method. Does not return {@code "<init>"} or {@code
+     * "<clinit>"} as ExecutableElement.getSimpleName() does.
+     *
+     * @param element a method declaration
+     * @return a user-friendly name for the method
+     */
+    public static CharSequence getFriendlyName(ExecutableElement element) {
```

Comment:
I agree this name isn't great.  I think `getUserFriendlyName` would be an improvement.  Here are a few other, mostly bad names that might spark an idea of a better name:

* `getUserFriendlySimpleName`
* `getUserFriendlyName`
* `getPlainSimpleName`
* `getUserUnderstandableSimpleName`
* `getPrintableSimpleName`
* ` getLegibleSimpleName`

Reply:
I like `getUserFriendlyName` best among these suggestions (and it's better than the current name).
Another idea is:
 * `getSimpleNameOrDescription`

Reply:
I like `getSimpleNameOrDescription` best.